kythe/go/extractors/bazel/treeset/treeset.go

```diff
@@ -0,0 +1,89 @@
+// Package treeset provides functions for extracting targets that use bazel treesets.
```

Comment:
This file needs a "Kythe Authors" copyright header.

Reply:
Of course it does... Done.